cmd/snap-update-ns/utils_test.go

```diff
@@ -248,3 +250,173 @@ func (s *realSystemSuite) TestSecureMkdirAllForReal(c *C) {
 	c.Check(fi.IsDir(), Equals, true)
 	c.Check(fi.Mode().Perm(), Equals, os.FileMode(0750))
 }
+
+// secure-mkfile-all
+
+// Ensure that we refuse to create a file with an relative path.
+func (s *utilsSuite) TestSecureMkfileAllRelative(c *C) {
+	err := update.SecureMkfileAll("rel/path", 0755, 123, 456)
+	c.Assert(err, ErrorMatches, `cannot create file with relative path: "rel/path"`)
+	c.Assert(s.sys.Calls(), HasLen, 0)
+}
+
+// Ensure that we can "create" the root directory.
```

Comment:
This comment doesn't seem to go with the test.

Reply:
I meant that ironically but I re-worded this now